wurst/_items/BattleglovesCritical.wurst

```diff
@@ -0,0 +1,89 @@
+package BattleglovesCritical
+
+import AbilityObjEditing
+import ObjectIdGenerator
+import ObjectIds
+import HashMap
+import ClosureEvents
+import ClosureTimers
+import DamageType
+import ID
+import AbilityObjectCreationUtils
+
+let CRITICAL_ABILITY_ID = compiletime(ABIL_ID_GEN.next())
+let SPELLBOOK_ABILITY_ID = compiletime(createHiddenSpellbookWithAbilities(toRawCode(CRITICAL_ABILITY_ID)))
+let COOLDOWN = 6.
+let COOLDOWN_SKIP_PER_ATTACK = 1.
+let MULTIPLIER = 1.6
+let CLOCK_TICK_INTERVAL = 0.05
+
+@compiletime function createCriticalAbility()
+    new AbilityDefinitionBladeMasterCriticalStrike(CRITICAL_ABILITY_ID)
+    ..setItemAbility(false)
+    ..setHeroAbility(false)
+    ..presetDamageMultiplier(lvl -> MULTIPLIER)
+    ..presetDamageBonus(lvl -> 0)
+    ..presetChancetoCriticalStrike(lvl -> 100)
+    ..presetChancetoEvade(lvl -> 0)
+    ..presetCooldown(lvl -> 0)
+    ..presetNeverMiss(lvl -> true)
+
+class BattleGlovesInstance
+    private unit wearer
+    private real lastCritTimestamp
```

Comment:
I think you don't need this. Just check `baseClock > COOLDOWN` and reset it to `0`.